### PR TITLE
Splitting off spec2vec

### PR DESCRIPTION
**Pull request by jspaaks**

### CODE REVIEW

**Comment by jspaaks** [comment]
sonarcloud fails due to missing token; we should fix this after we transfer the repo to iomega

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+1. you have a question;
+1. you think you may have found a bug (including unexpected behavior);
+1. you want to make some kind of change to the code base (e.g. to fix a bug, to add a new feature, to update documentation);
+1. you want to make a new release of the code base.
+
+The sections below outline the steps in each case.
+
+## You have a question
+
+1. use the search functionality [here](https://github.com/matchms/matchms/issues) to see if someone already filed the same issue;
```
Comment (fdiblen) [inline CONTRIBUTING.md]:
```suggestion
1. use the search functionality [here](https://github.com/iomega/spec2vec/issues) to see if someone already filed the same issue;
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+1. if your issue search did not yield any relevant results, make a new issue;
+1. apply the "Question" label; apply other labels when relevant.
+
+## You think you may have found a bug
+
+1. use the search functionality [here](https://github.com/matchms/matchms/issues) to see if someone already filed the same issue;
```
Comment (fdiblen) [inline CONTRIBUTING.md]:
```suggestion
1. use the search functionality [here](https://github.com/iomega/spec2vec/issues) to see if someone already filed the same issue;
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+1. Check author list in `citation.cff` and `.zenodo.json` files
+1. Bump the version using `bump2version <major|minor|patch>`. For example, `bump2version major` will increase major version numbers everywhere its needed (code, meta, etc.) in the repo.
+1. Update the `CHANGELOG.md` to include changes made
+1. Goto [GitHub release page](https://github.com/matchms/matchms/releases)
+1. Press draft a new release button
+1. Fill version, title and description field
+1. Press the Publish Release button
+
+A GitHub action will run which will publish the new version to [anaconda](https://anaconda.org/nlesc/matchms).
```
Comment (fdiblen) [inline CONTRIBUTING.md]:
```suggestion
A GitHub action will run which will publish the new version to [anaconda](https://anaconda.org/nlesc).
```

**Review by fdiblen — APPROVED** [review]
:+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (jspaaks) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (jspaaks) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 3 above)
Comment (jspaaks) [reply CONTRIBUTING.md]: done

**Pull request merged**